### PR TITLE
Handle error

### PR DESCRIPTION
**Pull request by vjt960**
Set up the login to only route back to the home page upon successful login. If a login attempt is unsuccessful, it remains on the login page and displays the error on the form saying that the email or password was invalid instead of an alert. 

The same process works for the signup form. If a user attempts to sign in with an email that is already on record, an error will throw and a notification will alert that an account with that email already exists.

### CODE REVIEW

File: src/containers/LoginForm.js [hunk 1]
```diff
@@ -18,12 +19,17 @@ class LoginForm extends Component {
...
-    let user = await fetchUser(email, password);
-    if (user) {
+    try {
+      let user = await fetchUser(email, password);
       this.props.signIn(user);
+      this.props.history.push('/');
+      this.props.hasErrored('');
+    } catch ({ message }) {
+      this.props.hasErrored(message);
+      this.props.history.push('/login');
```
Comment (David5280) [inline src/containers/LoginForm.js]: Everything is looking great @vjt960!  I'm curious about this.props.hisotry.push('./login), does this store the history of the pages they've viewed for redirection purposes?

Comment (vjt960) [reply src/containers/LoginForm.js]: **I tried making it work with the NavLink, but...**

Comment (David5280) [reply src/containers/LoginForm.js]: Gotcha, well it seems to work well, I just haven't seen that before!  So...

Comment (TaylorNoelJordan) [reply src/containers/LoginForm.js]: Very interesting! Would you explain that to me more in depth the next time we're in person?

**Pull request merged**